src/misc/SvgPolyfill.ts

```diff
@@ -0,0 +1,11 @@
+import { InitializationEvents } from '../events/InitializationEvents';
+import { $$ } from '../utils/Dom';
+require('svg4everybody');
```

Comment:
Does this polyfill override the standard/native implementation in browsers, or does it only do it's job when there is a missing functionnality ?